.github/workflows/checkout-and-build-smithy/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
     - uses: actions/setup-java@v2
       with:
         java-version: 17
```

Comment:
Should this be 21 now?

Reply:
I don't think it really matters, but I can update it